spaceship/lib/spaceship/connect_api/users/users.rb

```diff
@@ -28,7 +28,7 @@ def delete_user(user_id: nil)
           users_request_client.delete("users/#{user_id}")
         end
 
-        # Change app permissions for user
+        # Add app permissions for user
         def add_user_visible_apps(user_id: nil, app_ids: nil)
```

Comment:
We've been prefixing these methods with the HTTP method so I think I'd like to stick to that if possible 😇  I realized I should have caught this `add_` sooner so maybe update that to be deprecated and change it 🤷‍♂️  My apologies on that!

```suggestion
        # @deprecated Use {#post_user_visible_apps} instead.
        def add_user_visible_apps(user_id: nil, app_ids: nil)
          post_user_visible_apps(user_id: user_id, app_ids: app_ids)
        end

        def post_user_visible_apps(user_id: nil, app_ids: nil)
```